docs/installation.rst

```diff
@@ -12,20 +12,11 @@ To install xclim via pip, run this command in your terminal:
 
     $ pip install xclim
 
-To install xclim with spatial subsetting tools (`clisops`_), ensure you have the necessary system dependencies installed and run:
```

Comment:
If we remove this section, we should probably also modify the `gis` recipe. Given that our `upstream` tox recipe is starting to get really complicated, maybe we can extract that from tox and offer a new recipe (that includes `flox` and `clisops`)?

Reply:
But the `[gis]` recipe doesn't exist.  haha, I forgot to mention it above, but it seems it was deprecated without updating this file!

Reply:
Haha, in that case, good call! I don't even recall when that happened.